internal/storage/url.go

```diff
@@ -1,63 +1,45 @@
 package storage
 
+import "fmt"
+
 type URL struct {
-	Urls   map[string]string
-	Record *Records
+	Urls map[string]string
 }
 
-var URLStore *URL
-
-func NewURL(fileName string) (*URL, error) {
-	record, err := (&Records{}).New(fileName)
-	if err != nil {
-		return nil, err
-	}
-
+func NewStorage() (*URL, error) {
 	url := &URL{
-		Urls:   make(map[string]string),
-		Record: record,
-	}
-
-	// Загрузка записей из файла в URL.Urls
-	if err := url.LoadFromFile(fileName); err != nil {
-		return nil, err
+		Urls: make(map[string]string),
 	}
 
 	return url, nil
 }
 
-func (s *URL) Push(key, value string) {
-	// Добавление записи в URL.Urls
-	s.Urls[key] = value
+func (u *URL) GetOriginalURL(shortURL string) (string, error) {
+	value, exists := u.Urls[shortURL]
 
-	// Создание новой записи и сохранение в файл
-	record := &Record{
-		UUID:        0, // Используйте соответствующий UUID
-		ShortURL:    key,
-		OriginalURL: value,
-	}
-
-	// Сохранение в файл
-	if err := s.Record.WriteEvent(record); err != nil {
-		panic(err) // Обработка ошибки по вашему усмотрению
+	if !exists {
+		return "", fmt.Errorf("shortURL not found")
```

Comment:
```suggestion
		return nil, fmt.Errorf("shortURL not found")
```
если возвращать ссылкой будет чуть удобней и правильней - т.к. пустая строка это тоже валидное значение

Reply:
func (u *URL) GetOriginalURL(shortURL string) (string, error)   string возвращаемый тип

Reply:
Я вижу, и в этом заключается вопрос - если возможна ситуация когда не смогли найти url - почему возвращаемый тип string  в случае ошибки? Пустая строка это валидное значение. Если значение не найдено, правильно возвращать nil

Reply:
как я в го могу указать тип string or null если в go нет такой возможности. Или я что-то упустил? И на сколько я вижу по обучению всегда проверяется валидность по error а не переменной значения

Reply:
Видимо упустил
```go
func (u *URL) GetOriginalURL(shortURL string) (*string, error) {
    return nil, fmt.Errorf("shortURL not found")
}
```